components/availability/src/Availability.svelte

```diff
@@ -35,6 +35,8 @@
   export let show_ticks: boolean = true;
   export let email_ids: string[] = [];
   export let allow_booking: boolean = false;
+  export let max_bookable_slots: number = 1;
+  export let participant_threshold: number = 1;
```

Comment:
Couple of questions about this prop:
 - Should we instead have users enter a decimal between 0 and 1? This seems like a more common convention for passing in a ratio / percentage.
 - Why do we default to 1%? Doesn't something like 50% make more sense as the default?
 - Can we look into renaming this prop into something a bit more descriptive of the value we expect to be passed in? Maybe something like `partial_timeslot_ratio`.

Reply:
Thanks @ozsivanov  -- changing this to a 0-1 decimal and renamed as `partial_bookable_ratio`

Default as 1: I've changed this to 0, but IMO it should either default to the minimum/0 (all partial events are bookable) or max/1 (no partial events are bookable) -- anything in the middle / 0.50 would lead to some mental overhead for someone who implements but doesn't have time to read docs.